Created scratch org with a custom alias does not get deleted at the end of the pipeline

When using the "Create / Delete a Scratch Org" task and specifying an alias that is not the default of "scratchorg" and setting maintainorg to "delete" the scratch org is not deleted in the automatically created "Post-job: Create scratch org" task.

It reports the following error:
```
ERROR running force:org:delete:  No org configuration found for name scratchorg
##[error]Command failed: npx sfdx force:org:delete -u  scratchorg -v MCDDevHubOrg -p
```
It appears to ignore the name of the scratch org created and looks for the default of **scratchorg**.

--- a/packages/azpipelines/BuildTasks/ManageScratchOrgTaskCurrent/PostManageScratchOrg.ts
+++ b/packages/azpipelines/BuildTasks/ManageScratchOrgTaskCurrent/PostManageScratchOrg.ts
@@ -8,13 +8,13 @@ async function run() {
     const action: string = tl.getInput("action", true);
     const maintainorg: string = tl.getInput("maintainorg", true);
     const devhub_alias: string = tl.getInput("devhub_alias", true);
-    const target_org: string = tl.getInput("target_org", true);
+    const alias: string = tl.getInput("alias", true);
 
     if (action == "Create" && maintainorg == "delete") {
       console.log("SFPowerScript.. Delete the created scratch org");
 
       let deleteScratchOrgImpl: DeleteScratchOrgImpl = new DeleteScratchOrgImpl(
-        target_org,
+        alias,
         devhub_alias
       );
       console.log("Generating Delete Scratch Org command");

